src/elements/content-explorer/ContentExplorer.js

```diff
@@ -1658,6 +1659,7 @@ class ContentExplorer extends Component<Props, State> {
         /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
         return (
             <Internationalize language={language} messages={messages}>
+                <BrandingStyles />
```

Comment:
i think change this to `CustomBranding` wrapper that either calls blueprint's `BrandingStyles` or use custom implementation. refer to EUA implementation